src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -574,6 +642,7 @@ const useGroupInfo = (channelId = null) => {
     blockModalRef,
     isFetchingAllowAnonDM,
     isLoadingInitChat,
+    isOpenModalChangeName,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `isOpenModalChangeName` has been added to the returned object. Make sure this is used properly in the component where this hook is used.
```diff
+ isOpenModalChangeName,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->